src/modules/user/user.service.ts

```diff
@@ -155,7 +162,10 @@ export class UserService {
     }
   }
 
-  async setAvatar(file: Express.Multer.File, data: { userId: string }): Promise<string> {
+  async setAvatar(
+    file: Express.Multer.File,
+    data: { userId: string },
```

Comment:
Lyudmyla, I can't answer your question, cause this part (working with avatar) was made by Artur. I only fixed the error return in catch block. This part of code is highlited, because my prettier formatted it

Reply:
sync with Artur, if I will approve your PR it would never be fixed

Reply:
synced with Artur and fixed data type